### ISSUE TITLE
teraslice external modules

### ISSUE THREAD

**Issue by jsnoble** [terascope/teraslice#3649]
- typically we put our approved helper functions and libraries come out of our `utils` library. We have several exported function from lodash here that we allow. However in teraslice we directly import lodash into use but they should live in utils if they are universal (which several of them are).  Some of these helper function are not needed since we now have more modern tools like Object.entries and the like. We should consolidate the functions we need from teraslice into utils. Many of these functions also live in the k8s code section so we need to tests there

**Comment by godber** [comment]
Is this issue closed by the following PR:

https://github.com/terascope/teraslice/pull/3863

**Comment by jsnoble** [comment]
yes this one is completed, however there are more libraries that probably need to be consolidated, but since this ticket is specific about lodash in teraslice we can close it
